transformations/propbank_srl_roles/transformation.py

```diff
@@ -0,0 +1,71 @@
+from allennlp.predictors.predictor import Predictor
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from transformations.propbank_srl_roles.findargs import perturb_text
+
+
+def srl_args(text, max_outputs=1):
+    predictor = Predictor.from_path(
```

Comment:
You should load this in the constructor. This will load the model for every sentence.